lib/govuk_test.rb

```diff
@@ -1,5 +1,23 @@
 require "govuk_test/version"
 
+require "capybara"
+require "puma"
```

Comment:
Ooh does this run the app on Puma rather than Unicorn?

If so do you know if it's multi-threaded as that could cause some surprises.

Reply:
@rubenarakelyan, do you remember why https://github.com/alphagov/collections/pull/774 needed Puma?

Reply:
Capybara was upgraded from 2 to 3 which now uses puma as its default web server for serving the app to be tested rather than the rails built-in Webrick server. Unicorn can be used but via the `rack-handlers` gem which hasn't been upgraded in a long time, and I'd rather not introduce another dependency. Is there an issue with using puma?